scripts/storybook/webpack.config.js

```diff
@@ -74,7 +74,12 @@ module.exports = (/** @type {webpack.Configuration} */ config) => {
 
   config.resolve.extensions.push('.ts', '.tsx');
 
-  config.resolve.alias = getResolveAlias();
+  config.resolve.alias = {
+    ...getResolveAlias(),
+    '@fluentui/examples$': path.join(__dirname, '../../packages/examples/src'),
```

Comment:
Not super important but it's better to use `path.resolve` to get a path without the `../`